packages/helper-settings/index.js

```diff
@@ -1,7 +1,6 @@
+/* global browser */
```

Comment:
Have you tried to import the polyfill right here? I think this should work and so it will be part of the webpack build.

Reply:
I don’t not try that, just followed the instructions from the readme. But I’ll give it a try and report back, thanks!

Reply:
Yes, I think you'll need to import it here, (instead of referencing it inside `node_modules` [above](https://github.com/OctoLinker/OctoLinker/pull/466/files#diff-76e85634924729360ea8e5952981214dR20)) to make sure it's present in the build delivered to end users (rather than just the development version).